provider/resource.go

```diff
@@ -1749,7 +1751,8 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_purview_account": {Tok: azureResource(azurePurview, "Account")},
 
 			// Management
-			"azurerm_management_group_template_deployment": {Tok: azureResource(azureManagement, "GroupTemplateDeployment")},
+			"azurerm_management_group_template_deployment":      {Tok: azureResource(azureManagement, "GroupTemplateDeployment")},
+			"azurerm_management_group_subscription_association": {Tok: azureResource(azureManagement, "GroupSubscriptionAssociate")},
```

Comment:
Why `Associate` vs. `Association`?

Reply:
Because typo :D I will fix it up before we merge it - any other issues with this one?